common/src/main/java/com/example/android/uamp/media/extensions/MediaMetadataCompatExt.kt

```diff
@@ -124,7 +124,7 @@ inline val MediaMetadataCompat.downloadStatus
  * Custom property for storing whether a [MediaMetadataCompat] item represents an
  * item that is [MediaItem.FLAG_BROWSABLE] or [MediaItem.FLAG_PLAYABLE].
  */
-@MediaItem.Flags
+//@MediaItem.Flags
```

Comment:
? Should this be removed?

Reply:
It was causing a compilation error: `error: incompatible types: NonExistentClass cannot be converted to Annotation
    @error.NonExistentClass()` and things seem to run fine without it so I'm guessing it can be removed 🤷